test/addons/rook-cluster/kustomization.yaml

```diff
@@ -4,28 +4,30 @@
 # yamllint disable rule:line-length
 ---
 resources:
-  - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/cluster-test.yaml
-patchesJson6902:
-  - target:
-      kind: CephCluster
-      name: my-cluster
-      namespace: rook-ceph
-    patch: |-
-      # Minikube does not persist /var/lib/rook, but it persists /data/*
-      # https://minikube.sigs.k8s.io/docs/handbook/persistent_volumes/#a-note-on-mounts-persistence-and-minikube-hosts
-      - op: replace
-        path: /spec/dataDirHostPath
-        value: /data/rook
-      # Enable host networking - ceph monitors will be available on the host
-      # network, exposed outside of the cluster.
-      - op: add
-        path: /spec/network
-        value:
-          provider: host
-      # Enable log collection. Logs are created in /data/rook/rook-ceph/log and rotated.
-      - op: add
-        path: /spec/logCollector
-        value:
-          enabled: true
-          periodicity: daily
-          maxLogSize: 50M
+- https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/cluster-test.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
These are better located at the top to we can group together the keys that make a difference.